playground/mocks/config/responseConfig.js

```diff
@@ -8,6 +8,10 @@ const idx = {
     'well-known-openid-configuration'
   ],
 
+  '/oauth2/default/v1/token': [
```

Comment:
nit: comment out this change to response config ?

Reply:
Done